probcal/models/gaussian_nn.py

```diff
@@ -30,8 +30,8 @@ def __init__(
         self,
         backbone_type: Type[Backbone],
         backbone_kwargs: dict,
-        optim_type: OptimizerType,
-        optim_kwargs: dict,
+        optim_type: OptimizerType | None = None,
+        optim_kwargs: dict | None = None,
```

Comment:
I don't see anywhere that we handle the `None` case. Is that accounted for somewhere?

I think this is one of the reasons I chose to use the monolith config setup in the DDPN code. But really, this should be optional if we're just instantiating a model for evaluation

Reply:
Yes, trying to enable models to be used easily without training. I can add and `Optional` tag